desktop/NodeManager.ts

```diff
@@ -236,7 +236,7 @@ class NodeManager {
 
   connectToRemoteNode = async (apiUrl?: SocketAddress | PublicService) => {
     this.nodeService.createService(apiUrl);
-    return this.getNodeStatus(5);
+    return this.updateNodeStatus();
```

Comment:
internally , will do this.getNodeStatus(5) and will send  this.sendNodeStatus(status); to the client side